http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesServiceDbQueries.scala

```diff
@@ -113,6 +116,32 @@ object KubernetesServiceDbQueries {
     joinFullAppAndUnmarshal(kubernetesClusterQuery.findByIdQuery(id), nodepoolQuery, appQuery)
       .map(_.headOption)
 
+  def markPendingCreating(nodepoolId: NodepoolLeoId, appId: AppId, clusterId: Option[CreateCluster])(
```

Comment:
shouldn't `clusterId` be something like `Option[ClusterId]`

Reply:
ah, old param name. Renamed it to something more fitting. The `createCluster` ADT is needed, since creating a cluster involves creating a default nodepool along with it

Reply:
don't you also have `nodepoolId` passed in already? or that's a different `nodepoolId`?